proposals/aip-1.md

```diff
@@ -0,0 +1,128 @@
+
+## Proposal Statuses ##
+The possible paths of the status of AIPs are as follows:
+
+  Draft -> Accepted -> Final
+  Draft -> Withdrawn
+  Draft -> Rejected
+  Draft -> Deferred
+  Final -> Replaced
+
+Unlike the EIP process, Informational and Process AIPs do not have the status "Active" and will have the status Final when they are accepted and the board is expected to execute the proposals on behalf of the community.
+
+
+# Proposal Structure: What belongs in a successful Proposal? #
+
+# Formats and Templates #
+
+# Proposal Header Preamble #
+
+# Auxiliary Files #
+
+# Transferring Proposal Ownership #
+
+# Proposal Editors #
+
+# Proposal Editor Responsibilities and Workflow #
+
+# History #
+
+# Copyright #
+
+
+# Assumptions #
+- Assumes 2FA is required by the GitHub org and all directors on the Board of Directors are using 2FA when submitting PRs, commits, and reviewing proposals.
+- **Roles:**
+  - ONLY the [@Agorex-io/board-of-directors](https://github.com/orgs/Agorex-io/teams/board-of-directors) can approve proposals that move beyond a PR and end up merged to the repository. 
+  - [@Agorex-io/proposal-editors](https://github.com/orgs/Agorex-io/teams/proposal-editors) cannot approve, but can facilitate the process and guide champions and authors. They can also edit PRs in progress to help get the preamble and formatting correct according to the workflow. 
+  
+- Assumes [required reviews have been set up **to include repository admins**](https://help.github.com/articles/enabling-required-reviews-for-pull-requests) option is on.
+- Assumes [Dismiss stale pull request approvals when new commits are pushed](https://help.github.com/articles/enabling-required-reviews-for-pull-requests/) option is on.
+- Assumes that **Board of Directors** is how we refer to the governing body for the community.
+  - I'm using "Board of Directors" as we have referred to it this way most often (and in AIP-2 issue discussions).
+- Assumes the target size for the Board of Directors is 9.
+- Note that the EIP process in EIP-1 is written focused on proposals for featurs and feels retrofitted for process and informational proposals. I am trying to make this more designed around community/structural proposals first, and expect we'll add a "feature proposal" type later. I think this will be better basis for the process in our case (which is decidedly different circumstance than Ethereum which also maintains a protocol with many clients & implementations).
+
+
+# Open Issues / TODO #
+- [ ] Near the mention of "We use GitHub's [required reviews for pull requests]" above, but that is for every review/pull of the PR. PRs can be merged in a Draft status without being accepted. More final status (Accepted, Deferred, Rejected, and Replaced, Final) require more careful reviews. That needs clarified. Or we could make a different directory for Draft proposals and Accepted/Finalized/Active/Replaced proposals to keep the review/approval requirement lower for Draft proposals.
```

Comment:
Using different branches for this as long as it is well documented is one option.

I may be misunderstanding your question though.

When should members submit a review that "approve"s a PR? Only when we feel it should be Accepted?

Reply:
This was more of a note to myself, and I think is now resolved with the "Acceptance Process" section. I will remove this and we can discuss the Acceptance Process section now.